msteams-platform/TOC.yml

```diff
@@ -3,566 +3,568 @@
 - name: What's new?
   href: whats-new.md
 - name: Glossary
-  href: get-started/glossary.md  
+  href: get-started/glossary.md
 - name: Overview
-  expanded: false 
+  expanded: false
   items:
-  - name: Teams app that fits
```

Comment:
It looks like your authoring environment inadvertently altered the whitespace in the source TOC file. Would it be possible to reapply your edits in a way that doesn't alter the rest of the file?

Reply:
not sure how to resolve this? is there a way to undo my commits done on this file and fix it starting from a clean slate?